ajax/libs/jquery.loadTemplate/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "jquery.loadTemplate",
+    "description": "A template plugin for jQuery. Allows templating without cluttering JavaScript code with markup",
```

Comment:
Let's let the `description` as same with upstream.
modify to `"description": "jQuery plugin for loading and using templates. The plugin is designed to be simple yet powerful, and supports loading templates from within the page, or using AJAX to load html files.",`
Here is the upstream's package.json file you could refer to: https://github.com/codepb/jquery-template/blob/master/package.json#L6
Thank you.